local-volume/provisioner/README.md

```diff
@@ -0,0 +1,53 @@
+# local-volume-provisioner
+
+NOTE: provisioner is still in development and is not functional yet
+
+local-volume-provisioner is an out-of-tree static provisioner for the Local volume plugin, which is a 1.7 alpha feature.
+
+It runs on each node in the cluster and monitors specified directories to look for new local mount points.  It then statically creates a Local PersistentVolume for each mount point.  It also monitors when the PersistentVolumes have been deleted, and will clean up the volume.
```

Comment:
thanks, can you also update the doc.  The deletion flow basically works, but is not quite intuitive since admin will have to find the PVC bound to PV, and delete the PVC.  I think this is the right model, but just want to make sure this is clear to users.